settings.gradle.kts

```diff
@@ -94,6 +94,7 @@ include(
 include(
     ":feature:mail:account:api",
     ":feature:mail:folder:api",
+    ":feature:mail:messages",
```

Comment:
I prefer a singular form for the module name, it fits better to a `:feature:mail:message:detail`, `:feature:mail:message:list` naming scheme.